os/tools/mksamsungheader.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+############################################################################
+#
+# Copyright 2021 Samsung Electronics All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+############################################################################
+import os
+import sys
+
+TOOL_DIR = '/root/tizenrt/os/tools/'
+############################################################################
+#
+# This script generates samsung binary header for kernel or user.
+#
+# 1. Kernel binary header information :
+#
+# total header size is (14 + #)bytes.
+# +----------------------------------------------------------------------------------------------
+# |  CRC32   | Header size | Binary Version |  Binary Size | Secure Header Size | Secure Header |
+# | (4bytes) |   (2bytes)  |    (2bytes)    |   (4bytes)   |      (2bytes)      |    (#bytes)   |
+# +----------------------------------------------------------------------------------------------
+#  * A CRC32 is checksum value for the entire binary from Header size to the end of binary.
+#  * A size of Secure Header depends on board.
+#
+#
+# 2. User binary header information on APP_BINARY_SEPARTION
```

Comment:
This file does not make the header of user binary. No need to show this here. Let's remove this block.